azure-iot-device/azure/iot/device/provisioning/internal/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""Azure Provisioning Device Internal
```

Comment:
>Azure Provisioning Device Internal [](start = 3, length = 34)

Not sure that a grab-bag package of things deemed "internal" is the best idea. Furthermore, everything we don't expose through the API is already considered internal.

Reply:
I could not find a good name and considered internal as the temp name. Open to changing the name.

---
In reply to: [277836013](https://github.com/Azure/azure-iot-sdk-python-preview/pull/81#discussion_r277836013) [](ancestors = 277836013)

Reply:
I'm not sure polling applies to everything here, which I guess is kind of the point - nothing about this package is really stuff that all goes together. It's not just the name, it's the fact that these modules are not exactly semantically related enough to form a package that could have a meaningful name.

---
In reply to: [281379996](https://github.com/Azure/azure-iot-sdk-python-preview/pull/81#discussion_r281379996) [](ancestors = 281379996,277836013)

Reply:
open to replacement of files in different folders...however i want some suggestion...they can go in transport..is that a good idea for now 

---
In reply to: [282608299](https://github.com/Azure/azure-iot-sdk-python-preview/pull/81#discussion_r282608299) [](ancestors = 282608299,281379996,277836013)

Reply:
> open to replacement of files in different folders...however i want some suggestion...they can go in transport..is that a good idea for now
> 
> In reply to: [282608299](https://github.com/Azure/azure-iot-sdk-python-preview/pull/81#discussion_r282608299) [](ancestors = 282608299,281379996,277836013)

@BertKleewein any suggestions? I honestly don't know enough about this to make good suggestions. I guess polling is alright, if a couple things get moved out (see other comments)